doc/spinel-protocol-src/spinel-prop-core.md

```diff
@@ -239,6 +245,59 @@ Unlike most other properties, setting this property to true when the
 value of the property is already true **MUST** fail with a last status
 of `STATUS_ALREADY`.
 
+### PROP 10: PROP_HOST_POWER_STATE {#prop-host-power-state}
+
+* Type: Read-Write
+* Packed-Encoding: `C`
+* Default value: 4
+
+Octets: |        1
+--------|------------------
+Fields: | `HOST_POWER_STATE`
+
+Describes the current power state of the *host*. This property is used
+by the host to inform the NCP when it has changed power states. The
+NCP can then use this state to determine which properties need
+asynchronous updates. Enumeration is encoded as a single unsigned
+byte. These states are defined in similar terms to `PROP_POWER_STATE`
+((#prop-power-state)).
+
+Defined values are:
+
+*   0: `HOST_POWER_STATE_OFFLINE`: Host is physically powered off and
+    cannot be woken by the NCP. All asynchronous commands are
+    squelched.
+*   1: `HOST_POWER_STATE_DEEP_SLEEP`: The host is in a low power state
+    where it can be woken by the NCP but will potentially require more
+    than two seconds to become fully responsive.
```

Comment:
Agreed with the assumption, but the wording reads as if the NCP will have a timeout of X (2 seconds?) before the host becomes fully responsive.  Given the wide variety of systems, I don't think we should document an explicit timeout as it suggests that after the timeout expires the NCP can try to communicate. Rather we should require that when the host exits this sleep state it must be the first to communicate with either an explicit PROP_POWER_STATE message or any other message if available at the time.